var/spack/repos/builtin/packages/miopen-hip/package.py

```diff
@@ -0,0 +1,44 @@
+# Copyright 2013-2020 Lawrence Livermore National Security, LLC and other
+# Spack Project Developers. See the top-level COPYRIGHT file for details.
+#
+# SPDX-License-Identifier: (Apache-2.0 OR MIT)
+
+
+from spack import *
+
+
+class MiopenHip(CMakePackage):
+    """AMD's library for high performance machine learning primitives."""
+
+    homepage = "https://github.com/ROCmSoftwarePlatform/MIOpen"
+    url = "https://github.com/ROCmSoftwarePlatform/MIOpen/archive/rocm-3.5.0.tar.gz"
+
+    maintainers = ['srekolam', 'arjun-raj-kuppala']
+
+    version('3.5.0', sha256='aa362e69c4dce7f5751f0ee04c745735ea5454c8101050e9b92cc60fa3c0fb82')
+
+    variant('build_type', default='Release', description='CMake build type')
```

Comment:
What possible values does this have?

Reply:
> What possible values does this have?

Release & debug variants available, I have added the same.